wifiphisher/common/phishinghttp.py

```diff
@@ -78,16 +123,23 @@ def post(self):
             terminate = True
 
 
-def runHTTPServer(ip, port, ssl_port, t):
+def runHTTPServer(ip, port, ssl_port, t, em):
     global template
     template = t
+
+    # Get all the UI funcs and set them to uimethods module
+    for f in em.get_ui_funcs():
+        setattr(uimethods, f.__name__, f)
+
     app = tornado.web.Application(
         [
-            (r"/.*", CaptivePortalHandler)
+            (r"/validate/.*", ValidateHandler, {"em": em}),
```

Comment:
While in most cases an extension will use these methods to validate captured creds, there may be other cases too (e.g. we may want to see if a user exists in a system but not validate his password).

Maybe a "/backend/" and "BackendHandler" are more appropriate names.